test/ejabberd_tests/tests/cluster_commands_SUITE.erl

```diff
@@ -110,7 +116,59 @@ one_to_one_message(ConfigIn) ->
                 Stanza2 = escalus:wait_for_stanza(Alice, 5000),
                 escalus:assert(is_chat_message, [<<"Oh hi!">>], Stanza2)
         end).
+%%--------------------------------------------------------------------
+%% Ejabberdctl tests
+%%--------------------------------------------------------------------
+
+set_master_test(ConfigIn) ->
+    TableName = passwd,
+    NodeList = nodes(),
+    ejabberdctl("set_master", ["self"], ConfigIn),
+    [MasterNode] = rpc_call(mnesia, table_info, [TableName, master_nodes]),
+    true = lists:member(MasterNode, NodeList),
+    RestNodesList = lists:delete(MasterNode, NodeList),
+    OtherNode = hd(RestNodesList),
+    ejabberdctl("set_master", [atom_to_list(OtherNode)], ConfigIn),
+    [OtherNode] = rpc_call(mnesia, table_info, [TableName, master_nodes]),
+    ejabberdctl("set_master", ["self"], ConfigIn),
+    [MasterNode] = rpc_call(mnesia, table_info, [TableName, master_nodes]).
 
 %%--------------------------------------------------------------------
 %% Helpers
 %%--------------------------------------------------------------------
+
+ejabberdctl(Cmd, Args, Config) ->
```

Comment:
Since these are copied from ejabberdctl_SUITE and almost unchanged, I think it would be nice to move them to new helper module and make both suites use the helper mod.